modules/core/04-channel/keeper/handshake.go

```diff
@@ -200,11 +200,19 @@ func (k Keeper) WriteOpenTryChannel(
 	k.SetNextSequenceRecv(ctx, portID, channelID, 1)
 	k.SetNextSequenceAck(ctx, portID, channelID, 1)
 
+	channelBefore, found := k.GetChannel(ctx, portID, channelID)
+	var prevState string
+	if found != true {
+		prevState = "None"
+	} else {
+		prevState = channelBefore.State.String()
+	}
```

Comment:
I think you can remove this and revert to hard-coding `types.UNINITIALIZED.String()` in the log arg.
Afaik crossing hellos were removed from the open TRY handshake handler in v4 so the only valid previous state is `UNINITIALIZED`.

Reply:
My bad. Ty <3